packages/frontend/src/Layout.tsx

```diff
@@ -1,9 +1,11 @@
 import Container from '@mui/material/Container';
 import { Outlet } from 'react-router-dom';
 
+import { AppBarComponent } from './components/AppBar';
```

Comment:
```suggestion
import { AppBarComponent } from 'components/AppBar';
```
There is a base url inside tsconfig (src folder), so you can directly import the component like above.

See [baseUrl documentation](https://www.typescriptlang.org/tsconfig#baseUrl) for more information